src/front/glsl/variables.rs

```diff
@@ -100,6 +100,25 @@ impl Program<'_> {
                 },
                 BuiltIn::Position,
                 true,
+                PrologueStage::empty(),
+            ),
+            "gl_FragCoord" => add_builtin(
+                TypeInner::Vector {
+                    size: VectorSize::Quad,
+                    kind: ScalarKind::Float,
+                    width: 4,
+                },
+                BuiltIn::Position,
+                false,
+                PrologueStage::FRAGMENT,
+            ),
+            "gl_FragDepth" => add_builtin(
+                TypeInner::Scalar {
+                    kind: ScalarKind::Float,
+                    width: 4,
+                },
+                BuiltIn::FragDepth,
+                true,
                 PrologueStage::FRAGMENT,
```

Comment:
I don't think that it can be read in the fragment stage has its only defined as `out`

Reply:
Right, I forgot that PrologueStage was about reading. I wrote this before we chatted about that.